packetbeat/_meta/fields.common.yml

```diff
@@ -175,6 +175,7 @@
         something like `GET /users/_search?name=test`. For MySQL, it is
         something like `SELECT id from users where name=test`.
 
+    #keyword(s)?
```

Comment:
@andrewkroh Could you take a look at the fields below. Potentially these were discussed in previous PR's.

Reply:
Good catch. These should both be set to keyword, to enable aggregations and fast indexing.

If full text search is deemed necessary by us or customers, `text` should be added as a multi-field.